### DISCUSSION TITLE
Introduce yourself!

### DISCUSSION THREAD

**Discussion by zpl-zak** [zpl-c/zpl#78] [General]
## Hi there!

This place allows you to discuss zpl related matters and open suggestions for the project!

* We're happy to listen to any ideas or considerations.
* Got anything cool to share? This is a great place to do so!
* Are you new here? Feel free to introduce yourself!

**Comment by Derailedzack** [comment]


**Reply by Derailedzack** [reply]
Most of this source is from the Client C Version.

**Reply by Derailedzack** [reply]
I'll move this to a new discussion.
